From fc0e20e4d4a43c86f9151f74164983bbe7a3a9c7 Mon Sep 17 00:00:00 2001
From: Andreas Stieger <astieger@suse.com>
Date: Sun, 25 Jan 2015 00:23:57 +0100
Subject: [PATCH] remove Debian specific command

Remove Debian specific architecture detection and replace with build time setting.

Signed-off-by: Stewart Smith <stewart@flamingspork.com>
---
 eatmydata.in | 4 +---
 1 file changed, 1 insertion(+), 3 deletions(-)

diff --git eatmydata.in eatmydata.in
index 40468db..e37f07c 100644
--- a/eatmydata.in
+++ b/eatmydata.in
@@ -15,9 +15,7 @@
 # You should have received a copy of the GNU General Public License
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
-export `dpkg-architecture|grep DEB_BUILD_MULTIARCH`
-
-shlib="/usr/lib/$DEB_BUILD_MULTIARCH/eatmydata.sh"
+shlib="@libexecdir@/eatmydata.sh"
 if [ -f "$shlib" ]; then
     . "$shlib"
 else
